### ISSUE TITLE
Reference to type 'IPAddress' claims it is defined in 'System.Net.Primitives', but it could not be found	

### ISSUE THREAD

**Issue by BvdVen** [lduchosal/ipnetwork#17]
When I update from version 2.03 tot 2.1.0 I get below error while building, on the System.Net.IPNetwork.Contains method.
> Reference to type 'IPAddress' claims it is defined in 'System.Net.Primitives', but it could not be found	

```
System.Net.IPAddress userIpAddress = System.Net.IPAddress.Parse(strUserIpAddress);
if (System.Net.IPNetwork.Contains(System.Net.IPNetwork.Parse(strIpRange), userIpAddress){....}
```

Is there are dependency missing during the update or am I doing something wrong?

**Comment by BigPacket** [comment]
=======================
Elliptic Curve Cryptography (ECC)	Key.

Another thing that was forgot when the update happened, was strong name.

**Comment by lduchosal** [comment]
Fixed in version 2.1.1
